app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +8,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".income" />
+        <activity android:name=".Income" />
```

Comment:
I changed income to Income to adhere with the class naming conventions of UpperCamelCase, this is the change in the android manfiest